### ISSUE TITLE
Gatsby enum ‘Duplicate Identifier’

### ISSUE THREAD

**Issue by ryanditjia** [dotansimha/graphql-code-generator#2025]
**Describe the bug**

Gatsby’s GraphQL server has enums that are separated with underscores. These underscores are removed during translation into TS enums by graphql-codegen. This removal results in ‘Duplicate Identifier’ TS errors for some field enums that are similar.

**To Reproduce**

Clone this minimal repo: https://github.com/ryanditjia/gatsby-graphql-code-generator

Run `npm start`, wait for server to spin up then `npm run gen`
See: [line 469](https://github.com/ryanditjia/gatsby-graphql-code-generator/blob/6b4516f8fab2a66ec1907cb4bfc0159bcf0df66d/src/types/graphql.ts#L469) and [529](https://github.com/ryanditjia/gatsby-graphql-code-generator/blob/6b4516f8fab2a66ec1907cb4bfc0159bcf0df66d/src/types/graphql.ts#L529) of the generated src/types/graphql.ts

**Expected behavior**

There should be no duplicates, perhaps the underscores should be kept as is? I’m having many more of these problems when using `gatsby-source-contentful`, due to Gatsby automatically creating reverse reference when using reference fields.

**Environment:**

- OS: 
- `@graphql-codegen/...`: 1.2.1
- NodeJS: 12.4.0

**Comment by dotansimha** [comment]
@ryanditjia By default, we remove underscores from enum values, in order to generate more readable enums. 
There is a flag that can help you fix that, but it has a bug, so I fixed it now :) 

Until we'll release 1.3.1, you can use the latest alpha version `1.3.1-alpha-96e1ad38.4`.

You can add the following configuration and it should work:

### Option 1 - Keep enum values as-is

```yml
config:
  namingConvention:
    enumValues: keep
    transformUnderscore: false
```

### Option 2 - Apply transformation, but keep underscores

```yml
config:
  namingConvention:
    transformUnderscore: false
```

**Comment by ryanditjia** [comment]
Great stuff, thanks!

**Comment by dotansimha** [comment]
Fixed in 1.3.1
